ruby-challenges.rb

```diff
@@ -10,6 +10,26 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+# create a method match_word that takes in an array and a string(letter)
+# the method then checks the array to contain the letter
+# the output is then a subset of the array with only the words containing that letter
+# tools and research: .include - returns true/false; .map iterates over the array but always returns the same array, filter returns a subset
+
+# takes in two arguments
+def match_word(array , string)
```

Comment:
Why did you use an empty array to store the array created by `filter`?

Reply:
That is a good question, the filter should return the values I put in the conditional -> i forgot to not use if statements in the filter! `def match_word(array , string)
    array.filter do |value|
        value.include? string
        end
    end
end`

Maybe the update would look something like that? Havent tested it, just thought about it.